<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClinicInfoDialog</class>
 <widget class="QDialog" name="ClinicInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clinic info</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetClinic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Наша поликлиника</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>174</width>
       <height>42</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Больница
Принстон-Плейнсборо</string>
     </property>
    </widget>
    <widget class="QLabel" name="clinicPic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>261</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainEditInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Наши специалисты</string>
    </attribute>
    <widget class="QComboBox" name="doctorSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLabel" name="doctorPhoto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>191</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="editDoctorDescription">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>261</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
